src/screens/FeedScreen/Header.js

```diff
@@ -312,9 +407,12 @@ const Header = ({
     height,
     source,
     headerStyle,
-    onHeaderOptionClicked: () => onHeaderOptionClicked(props),
-    hideThreeDot,
-    isPostDetail
+    isPostDetail,
+    isFollow,
+    onPressFollUnFoll,
+    onDeletePost,
+    isShowDelete,
+    isSelf
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The props passed to the `_renderProfileNormal` function have been updated to include the new props introduced in this PR. It's important to ensure that these props are used correctly within the `_renderProfileNormal` function.
```diff
+     isPostDetail,
+     isFollow,
+     onPressFollUnFoll,
+     onDeletePost,
+     isShowDelete,
+     isSelf
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->